README.md

```diff
@@ -103,6 +103,7 @@ The underlying cryptosystem is `x25519-xsalsa20-poly1305` which is implemented b
 | submissionId     | string | Unique response identifier, displayed as 'Response ID' to form respondents       |
 | encryptedContent | string | The encrypted submission in base64. |
 | created          | string | Creation timestamp.                 |
+| attachmentDownloadUrls | Record<string, string> | (Optional) Records containing field IDs and URLs where encrypted uploaded attachments can be downloaded. |
```

Comment:
could you also help to add the following to the README:
1. an example of how to call `downloadAndDecryptAttachments` in the sample code under "Usage"
2. a separate section which explains that attachments are fetched using presigned URLs, why this is secure, and a warning that we do not scan the attachments for malicious content

thanks! mentioning @liangyuanruo for any thoughts on this/other things to include

Reply:
@frankchn could you add the example under "Usage" as well! thanks!